src/kixi/hecuba/weather/service.clj

```diff
@@ -1,7 +1,22 @@
 (ns kixi.hecuba.weather.service
+  (:require [clojure.java.io :as io]
+            [clojure.tools.cli :refer [parse-opts]])
   (:gen-class))
 
-(defn -main
-  "I don't do a whole lot ... yet."
-  [& args]
-  (println "Hello, World!"))
+(def cli-options
+  [["-c" "--config CONFIG" "Path to config edn file."
+    :default "config/settings.edn"]])
+
+(defn load-config [filename]
+  (with-open [r (io/reader filename)]
+    (read (java.io.PushbackReader. r))))
+
+(defn find-entities [project-id api-endpoint entity-type]
+  )
+
+(defn -main [& args]
+  (let [{:keys [config] :as opts} (:options (parse-opts args cli-options))
+        edn-config (load-config config)]
+    (find-entities (:project-id edn-config)
+                   (:api-endpoint edn-config)
+                   (:entity-type edn-config))))
```

Comment:
Thanks Bruce, took a minute to wrap my head around it but I understand. 

`(find-entities edn-config edn-config edn-config)` is ` (find-entities project-id
                   api-endpoint
                   entity-type)`

Thanks for the heads up.